pkg/rke2/psp.go

```diff
@@ -324,36 +324,34 @@ func setPSPs() func(context.Context, <-chan struct{}, string) error {
 					if apierrors.IsNotFound(err) {
 						tmpl := fmt.Sprintf(nodeClusterRoleBindingTemplate, globalRestrictedRoleName)
 						if err := deployClusterRoleBindingFromYaml(ctx, cs, tmpl); err != nil {
-							logrus.Fatalf("psp: deploy psp: %s", err.Error())
+							logrus.Fatalf("psp: deploy clusterrolebinding: %s", err.Error())
 						}
 					} else {
-						logrus.Fatalf("psp: get clusterrole binding: %s", err.Error())
+						logrus.Fatalf("psp: get clusterrolebinding: %s", err.Error())
 					}
 				}
 			}
 
 			if err := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
-				if _, err := cs.CoreV1().Namespaces().Update(ctx, ns, metav1.UpdateOptions{}); err != nil {
-					if apierrors.IsConflict(err) {
```

Comment:
The initial update try and check for IsConflict are unnecessary as per [RetryOnConflict docs](https://godoc.org/k8s.io/client-go/util/retry#RetryOnConflict). Docs say you should always retrieve before changing/updating, and the retry handler will figure out on its own if the error is a conflict or not.